dummy/dummy.go

```diff
@@ -94,6 +99,22 @@ func (a *Analyzer) maxLineWidth(ch *lookout.Change) []*lookout.Comment {
 	return comments
 }
 
+func (a *Analyzer) noUAST(ch *lookout.Change) []*lookout.Comment {
```

Comment:
This seems like `maybeAddNoUastComment()`

Reply:
the naming is similar to the other methods like `maxLineWidth` or `lineIncrease`. Not every line exceeds max width or a file increase number of lines. The same here. Not every change don't have UAST.